resources/assets/js/annotations/components/annotationCanvas/polygonBrushInteraction.vue

```diff
@@ -25,13 +25,13 @@ export default {
         },
         isUsingPolygonFill() {
             return this.interactionMode === 'polygonFill';
-        },
+        }
```

Comment:
I prefer trailing commas.

```suggestion
        },
```